.vscode/settings.json

```diff
@@ -21,5 +21,8 @@
   "files.watcherExclude": {
     "**/.fvm": true
   },
-  "java.configuration.updateBuildConfiguration": "automatic"
+  "java.configuration.updateBuildConfiguration": "automatic",
+  "cSpell.words": [
```

Comment:
can we remove this from the PR? you should be able to put it in your own personal `settings.json`, but it does not need to be in the project file.